src/engine/IEngine.h

```diff
@@ -53,6 +55,18 @@ class IEngine
     virtual void restoreState( const EngineState &state ) = 0;
     virtual void setNumPlConstraintsDisabledByValidSplits( unsigned numConstraints ) = 0;
 
+    /*
+      Apply the stack to the newly created SmtCore, returns false if UNSAT is
+      found in this process.
+    */
+
```

Comment:
Remove empty line

Reply:
Looks like the comments should be swapped?

Reply:
Yep.. Swapped.